### DISCUSSION TITLE
Add stock model screenshots for applicable items in the CC rewards catalogue.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5778] [Suggestions]
Currently most if not all items in the shop have a preview image, and where they concern in-game rewards, they have a preview image that includes models/textures captured with the HD patch. As far as I know, the majority of players on CC do not play with this patch. Especially for rewards like the costumes, these rewards could look very different with the stock models, so I propose either another preview image be added for each applicable item to the shop, or that the existing images be replaced.

**Comment by Aethereus-ManaOnion** [comment]
Low cost and moderate impact solution. Overall I approve.
